inc/lib/LexemeParty.class.php

```diff
@@ -614,4 +614,4 @@ public static function displayRankings($rankings, $numberOfConcepts, $anchorLink
     
 }
 
-?>
```

Comment:
Can you please remove this change on the last line of the file?

Reply:
Huh, I don't know why it did that. I just used the built-in editor in GitHub, and didn't touch that line at all. 🧐

Reply:
It seems the editor automatically adds a newline at the end when you save, and there's no way (at least not that I found) to edit that in the web interface. But isn't newlines at the end the standard any way?

Reply:
Even if it's the standard, it should be done in another merge request. I took your commit locally, edited it, and pushed it.